example/feature_demo/demo_types.pb.gorm.go

```diff
@@ -49,17 +49,15 @@ import _ "github.com/golang/protobuf/ptypes/timestamp"
 var _ = fmt.Errorf
 var _ = math.Inf
 
-// TestTypesORM is a message that includes a representative sample of the
-// available types
 type TestTypesORM struct {
-	// Skipping field from proto option: ApiOnlyString
-	// The non-ORMable repeated field "Numbers" can't be included
-	OptionalString *string
-	BecomesInt     int32
-	// Empty type has no ORM equivalency
-	Uuid         *uuid.UUID `sql:"type:uuid"`
-	CreatedAt    time.Time
-	TypeWithIdId uint32
+	ANestedObjectTypeWithIDId uint32
+	BecomesInt                int32
+	CreatedAt                 time.Time
+	Numbers                   []int32
```

Comment:
Support for primitive type slices/arrays isn't supported yet, these fields will still need to be dropped until proper handling is added.

Reply:
Ok, fixed that.